controllers/kubevirtcluster_controller.go

```diff
@@ -141,6 +141,9 @@ func (r *KubevirtClusterReconciler) reconcileNormal(ctx *context.ClusterContext,
 		}
 	}
 
+	// Wait 10 seconds for LoadBalancer to assign external IP after been created
+	time.Sleep(10 * time.Second)
+
```

Comment:
instead of a sleep, can you push the key back onto the queue so the reconcile loop pops after 10 seconds. Then we just keep putting the key on the queue until we detect the ip of the external load balancer exists.

In general we dont' want to block the reconcile loop with a sleep.

Reply:
Totally agree, I have new changes ready for review, for the new code change, I have separated ClusterIP and LoadBalancer scenario, and will keep reconciling if LoadBalancer ExternalIP is not returned instead of return ClusterIP previously, Thanks